packages/redux/src/exchanges/selectors.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  getError,
+  getExchangeBookRequest as getExchangeBookRequestGetter,
+  getExchangeFilters as getExchangeFiltersGetter,
+  getIsLoading,
+  getResult as Result,
+} from './reducer.js';
+import type { ExchangesState } from './index.js';
+import type { StoreState } from '../index.js';
+
+/**
+ * Returns the exchanges result from the application state.
+ *
+ * @param state - Application state.
+ *
+ * @returns Returns exchanges result.
+ */
+export const getExchange = (state: StoreState) =>
+  Result(state.exchanges as ExchangesState);
+
+/**
+ * Returns the exchanges error.
```

Comment:
```suggestion
 * Returns the exchange error.
```

Since this selector will return the result of a single fetchExchange request, it should be named in the singular.

Reply:
done 👍